xarray/tests/test_cftimeindex_resample.py

```diff
@@ -228,7 +276,7 @@ def test_invalid_offset_error(offset: str | int) -> None:
     cftime_index = xr.date_range("2000", periods=5, use_cftime=True)
     da_cftime = da(cftime_index)
     with pytest.raises(ValueError, match="offset must be"):
-        da_cftime.resample(time="2D", offset=offset)  # type: ignore[arg-type]
+        da_cftime.resample(time="2h", offset=offset)  # type: ignore[arg-type]
```

Comment:
Switched to `"2h"`, since `"2D"` is no longer a `Tick`-like frequency.

Reply:
this seems like a major break. I've certainly used `"D"` for resampling to a daily frequency; and almost certainly used `"5D"` or `"7D"`. I think we should do a deprecation cycle.